build/php.m4

```diff
@@ -1338,7 +1338,7 @@ socklen_t x;
   ac_cv_socklen_t=no
 ]))
 if test "$ac_cv_socklen_t" = "yes"; then
-  AC_DEFINE(HAVE_SOCKLEN_T, 1, [Whether you have socklen_t])
+  AC_DEFINE(HAVE_SOCKLEN_T, 1, [Define to 1 if the system has the type socklen_t])
```

Comment:
Previous message was more typical?

Reply:
This is what Autoconf does, for example:

https://github.com/autotools-mirror/autoconf/blob/d3dcd5895d64f6c86275c9333e2760a867e540e7/lib/autoconf/types.m4#L169

However, it would be even better here to use this instead of defining another once used macro:

```
AC_CHECK_TYPES(socklen_t, [], [], [
  #ifdef HAVE_SYS_TYPES_H
  # include <sys/types.h>
  #endif
  #ifdef HAVE_SYS_SOCKET_H
  # include <sys/socket.h>
  #endif
])
```

Sending a patch into this pull request later on...

Reply:
So, now, the next commit does the same as before in a bit more "default" way :)